iree/base/api.cc

```diff
@@ -199,6 +199,40 @@ IREE_API_EXPORT int IREE_API_CALL iree_string_view_split(
   return offset;
 }
 
+static bool MatchPattern(absl::string_view value, absl::string_view pattern) {
```

Comment:
Nope, nor does any of the string stuff - I need to refactor the api file so that it's including other files instead (like iree/vm/api.h does). I haven't been able to decide on a naming scheme that makes it clear which files are C and which are C++ though; I may end up just moving everything but the API headers into iree/base/internal/ or something.